orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/pipeline/providers/gce/StatefullyUpdateBootImageStage.java

```diff
@@ -0,0 +1,59 @@
+/*
```

Comment:
This isn't really important, but the format of this header looks different than what we usually use (two `*` on every line).

Reply:
Oops, I need to get my copyright plugin squared away.